src/Extractor.php

```diff
@@ -81,10 +81,10 @@ public function run(Config $config, ?int $limit = null): void
                         $offset,
                         $limit,
                     );
-
-                    $this->userStorage->saveReport($report['name'], $data, $account['userId'], $primary);
                     $offset += $limit;
-                } while (count($data) > 0);
+                    $this->userStorage->saveReport($report['name'], $data, $account['userId'], $primary);
+                    usleep(5);
```

Comment:
Ten sleep je nutnej, jo?

Reply:
Co jsem testoval tak bez něj začalo seznam api vracet chybu na příliš velké množství requestů

Reply:
A nevrací to retry-after hlavičku nebo tak něco elegantnějšího?

Reply:
Vrací to chybovou hlášku s časem který se musí čekat, předělal jsem to

Reply:
Super, ještě ten test, ten už teď bude konzistentní? Jde mi o to aby se při každým PR nemusel updatovat, protože vrací pokaždé něco jiného?

Reply:
Test je konzistentní, musel jsem ho teď upravovat jen z důvodu množství dat aby to při testech neběželo moc dlouho a v testu pro debug jsem to musel zkrátit protože pak už neprošlel tak dlouhý výpis přes regex v assertu